docs/globals.rst

```diff
@@ -67,6 +67,7 @@ Currently, the following resources and properties are being supported:
       Layers:
       AutoPublishAlias:
       DeploymentPreference:
+      RolePath:
```

Comment:
We should remove this whole section and instead refer to the schema directly, as that’s the source of truth.

Reply:
Let's do it in a separate PR. Might have properties not in schema, or other interesting information.